README.md

```diff
@@ -1,33 +1,30 @@
 # Gameplay
 
-Issue tracker for Unvanquished gameplay related feedback, opinions, proposals
-and new ideas.
+Issue tracker for Unvanquished gameplay related feedback, problems and new ideas.
 
 To have a fruitful discussion, every issue here **must** fall clearly into one
 of the following categories:
 
-## Feedback
+## Experience & Feedback
 
 You can talk freely about how you felt during a game of Unvanquished or when
 playing in general. You can let us know what was frustrating or enjoyable to
 you but please make it clear that these are your feelings and refrain from
-including any opinions, proposals or ideas in such an issue. This way we can
-just accept your feelings as a fact and are free to pursue any number of
-solutions or improvements in another issue. If your feedback suggests that a
-particular thing needs to change in a particular way, we will close it.
+including any proposals or ideas in such an issue.
 
-Unlike opinions, feedback may not be answered with solution proposals but it
+Unlike `Problems`, `Experience & Feedback` are only for documentation purpose and not answered with solution proposals but it
 can help developers and players to come up with or agree with opinions about
-the current state of the game. Because it is inherently true and cannot be
-rejected, pure feedback can be very helpful and influencial in the long run.
+the current state of the game. 
+
+`Experience & Feedback` issues are closed 60 days after last activity. 
 
 **Example:** *When I play as a Dretch late in the match, I feel like I am stuck
 and can hardly evolve again.*
 
-## Opinion
+## Problem
```

Comment:
Agreed, as long as it remains clear that "problems" are perceived problems until there is agreement that something indeed is problematic.